### ISSUE TITLE
Use PatternStringProperty pattern instead of ScreenView.setScreenSummaryIntroAndTitle()

### ISSUE THREAD

**Issue by zepumph** [phetsims/joist#885]
From https://github.com/phetsims/joist/issues/873, this would be ideal to do, but in order to prevent regressions in dynamic behavior in the PDOM, we need to wait on this until we solve https://github.com/phetsims/scenery/issues/1442. So I'll mark as on hold.
